Cargo.toml

```diff
@@ -34,6 +34,7 @@ verbose = []
 "phy-raw_socket" = ["std", "libc"]
 "phy-tap_interface" = ["std", "libc"]
 "proto-ipv4" = []
+"fragmentation-ipv4" = ["proto-ipv4"]
```

Comment:
:bike: :house: : `ipv4-fragmentation`? Will it ever make sense to have a single generic `ip-fragmentation` feature? Is there a reason to keep two separate features for ipv4 and ipv6?

Reply:
I am not familiar with Ipv6 so I cannot comment on how similar to IPv4 it is. But it can be refactored in the future, if someone with more knowledge about Ipv6 can pitch in. I would keep it separate for now, better some fragmentation support than none:-)